bigflow/scaffold/templates/new-project/{{project_name}}/pyspark/workflow.py.jinja

```diff
@@ -0,0 +1,40 @@
+{% skip_file_unless pyspark_job -%}
+
+import functools
+import pathlib
+
+import bigflow
+import bigflow.dataproc
+import bigflow.resources
+
+from {{project_name}}.pyspark.config import workflow_config
+from {{project_name}}.pyspark.processing import run_pyspark_job
+
+
+pyspark_job = bigflow.dataproc.PySparkJob(
+    id='pyspark_job',
+
+    # Job entry point. First arguments of this function is 'bigflow.JobContext'
+    # Other arguments may be added by user via 'functools.partial' 
+    driver=functools.partial(
+        run_pyspark_job,
+        points=int(workflow_config.resolve_property('points')),
+        partitions=4,
+    ),
+
+    # Deploy options
+    bucket_id=workflow_config.resolve_property('staging_bucket'),
+    gcp_project_id=workflow_config.resolve_property('gcp_project_id'),
+    gcp_region=workflow_config.resolve_property('gcp_region'),
+    
+    # Use the same set of dependencies as '{{project_name}}'
+    pip_packages=bigflow.resources.get_resource_absolute_path('requirements.txt', pathlib.Path(__file__)),
+)
+
+
+pyspark_demo_workflow = bigflow.Workflow(
```

Comment:
Can you change the naming in this job to show what the job is doing? So here it's not `pyspark_workflow` but `pi_estimator`. Let's be consistent with the 2 other workflows.

Reply:
done